modules/end-user-guide/pages/url-parameter-for-starting-duplicate-workspaces.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: CONCEPT
+:navtitle: URL parameter for starting duplicate workspaces
+:description: URL parameter for starting duplicate workspaces
+:keywords: duplicate, duplicated, multiple
+// :page-aliases:
+
+[id="url-parameter-for-starting-duplicate-workspaces_{context}"]
+
+= URL parameter for starting duplicate workspaces
+
+Visiting a URL for starting a new workspace results in a new workspace according to the devfile and with a clone of the linked Git repository.
+
+In some situations, you may need to have multiple workspaces that are duplicates in terms of the devfile and the linked Git repository. You can do this by visiting the same URL for starting a new workspace with a URL parameter.
```

Comment:
```suggestion
In some situations, you may need to have multiple workspaces that are duplicated in terms of the devfile and the linked Git repository. You can do this by visiting the same URL for starting a new workspace with added URL parameter.
```

Reply:
Thank you, but I'm keeping `duplicates` here because `duplicated` has a somewhat different meaning. Also, `are duplicates` is sufficiently clear here to indicate that it's a noun and not a verb.